WikipediaUnitTests/Code/QuadKeyTests.swift

```diff
@@ -0,0 +1,57 @@
+import XCTest
+@testable import Wikipedia
+
+class QuadKeyTests: XCTestCase {
+
```

Comment:
A brief comment would be nice for future reference. This is a bit... inscrutable :)